terraform/variables.tf

```diff
@@ -0,0 +1,133 @@
+variable "project" {
+  default     = ""
+  description = "Project name (required for Premium SKU) - sds or cft. "
+}
+
+variable "environment_name" {
+  type = string
+}
+
+variable "resource_prefix"{
+  type    = string
+  default = ""
+}
+variable "namespace_prefix" {
+  type    = string
+  default = ""
+}
+
+variable "mssql_server" {
+  type    = string
+  default = ""
+}
+
+variable "topic_prefix" {
+  type    = string
+  default = ""
+  }
+
+variable "app_service_plan_sku_tier" {
+  type    = string
+  default = "Basic"
+}
+
+variable "app_service_plan_sku_size" {
+  type    = string
+  default = "B1"
+  }
+
+  variable "storage_account" {
+  type    = string
+  default = ""
+}
+
+variable "key_vault_name" {
+  type = string
+}
+
+variable "plan_prefix" {
+  type    = string
+  default = ""
+}
+
+variable "azure_sp_credentials_json" {
+  type    = string
+  default = null
+}
+
+variable "resource_group_name" {
+type        = string
+}
+
+variable "sku" {
+  type        = string
+  default     = "Standard"
+  description = "SKU type (Basic, Standard and Premium)"
+}
+
+variable "capacity" {
+  type        = number
+  default     = 0
+  description = "Specifies the capacity"
+}
+
+variable "function_app_name" {
+  type    = string
+  default = ""
+}
+
+variable "storage_account_name" {
+  type = string
+}
+
+variable "zone_redundant" {
```

Comment:
Needs to be prefixed with the resource it relates to

Reply:
done

Reply:
added prefix